frontend/src/pages/ActivityReport/index.js

```diff
@@ -37,8 +37,8 @@ const defaultValues = {
   topics: [],
 };
 
-const additionalNotes = 'this is an additional note';
-const approvingManagers = [2];
+const additionalNotes = '';
+const approvingManagers = [];
```

Comment:
Do we need these variables still? Seems like they can be removed and nothing passed on line 80?

Reply:
ahh good catch!